src/styles/blocks.scss/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px;
+    background-position: center;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-image: url(../images/star.svg);
+
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
The way you've implemented the active stars doesn't seem correct. The number of active stars should be determined by the modifier of the parent block, not by the position of the child elements. It would be more appropriate to use a loop in SCSS to generate the necessary classes for each possible number of active stars.